src/App.js

```diff
@@ -1,11 +1,77 @@
+import React, { useState } from "react";
 import './App.css';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import TodoApp from './components/TodoApp'
+import TodoList from './components/TodoList';
+import EditTodo from './components/EditTodo';
 
-function App() {
+const App = () => {
+
+  const [input, setInput] = useState('');
+  const [itask, setItask] = useState([]);
+  const [isEdit, setIsEdit] = useState(null);
+  const [toggleSubmit, setToggleSubmit] = useState(true);
+  const [error, setError] = useState(false);
```

Comment:
const [input, setInput] = useState('');
const [isEdit, setIsEdit] = useState(null);
const [toggleSubmit, setToggleSubmit] = useState(true);
const [error, setError] = useState(false);

these all should be in the EditTodo component. App component will only mange **itask** related stuff. @shivam-innotech